src/OXRS_WT32.h

```diff
@@ -32,7 +38,7 @@ class OXRS_WT32 : public Print
   void setMqttTopicPrefix(const char *prefix);
   void setMqttTopicSuffix(const char *suffix);
 
-  void begin(jsonCallback config, jsonCallback command);
+  void begin(jsonCallback config, jsonCallback command, climateUpdateCallback climateUpdate);
```

Comment:
Given this is an optional feature I would probably also remove this from `begin()` and add an explicit `setClimateUpdateCallback()` you can optionally use if the sensor is installed.

Reply:
I took the approach to have the sensor code in all versions. The shared I2C bus (with touch) is checked if  a SHT20 is connected. No action when no sensor found. This eleminates a bunch of #ifdefs in code which makes it better readable. The sensor related code is there anyway.

Reply:
Yeah I agree that is a good approach. Potentially becomes a problem when we add the next callback, and the next. 

But happy for you to merge this as is.